keps/sig-apps/sidecarcontainers.md

```diff
@@ -162,11 +193,29 @@ Init containers would be able to have `sidecar: true` applied to them as it's an
 
 Older Kubelets that don't implement the sidecar logic could have a pod scheduled on them that has the sidecar field. As this field is just an addition to the Container Spec the Kubelet would still be able to schedule the pod, treating the sidecars as if they were just a normal container. This could potentially cause confusion to a user as their pod would not behave in the way they expect, but would avoid pods being unable to schedule.
 
+## Design Details
 
-## Graduation Criteria
-
+### Test Plan
 //TODO
 
+### Graduation Criteria
+#### Alpha -> Beta Graduation
+* Addressed feedback from Alpha testers
+* Thorough E2E and Unit testing in place
+
+#### Beta -> GA Graduation
+* Sufficient number of end users are using the feature
+* No significant bugs found for 2 beta releases
```

Comment:
We generally assume that the alpha and beta periods are sufficient to *discover* the significant blocking bugs. We then say the GA criterion is that all known blocking bugs must be fixed. We don't usually add another "waiting period" after all known bugs are fixed, since that would make features very slow to progress.

Reply:
Ok, sounds reasonable, I saw this kind of thing in some other KEPs so I wasn't sure if it was convention to wait a few releases.